trying to fix broken <meta description=""> tag

gariepinus opened the pull request:
Vor meiner Änderung wurde für die Description folgender HTML Code generiert:

7:   <meta description="Darmstädter Erfahrungs-Austauschkreis des Chaos Computer Club, Hackspace und Hochschulgruppe an der TU Darmstadt
8:   ">

Die Newline nach dem eigentlichen Description-Text aber VOR dem schließenden "> hat zu invalidem HTML geführt. Hoffe mein Ansatz fixt das... (no idea what i am doing)

gariepinus committed:
trying to fix broken <meta description=""> tag

Vor meiner Änderung wurde für die Description folgender HTML Code generiert:

7:   <meta description="Darmstädter Erfahrungs-Austauschkreis des Chaos Computer Club, Hackspace und Hochschulgruppe an der TU Darmstadt
8:   ">

Die Newline nach dem eigentlichen Description-Text aber VOR dem schließenden "> hat zu invalidem HTML geführt. Hoffe mein Ansatz fixt das... (no idea what i am doing)

mweinelt commented:
Hey,

kannst du mal versuchen das bei dir mit jekyll zu bauen und dann anzuschauen?

Gruß Martin

gariepinus committed:
fix for meta-description tag

gariepinus committed:
html validation
- added lang info to html tag and removed (wrong) meta lang tag;
- removed unnessecary meta charset tag;

---

STATE: nach diesem commit sind alle HTML fehler im <head> entfernt. jekyll liefert valides HTML5 aus.
(allerdings gibt es immer noch ein paar Warnings um die man sich noch kümmern könnte)

gariepinus commented:
Hey,

habe das ganze jetzt mal gebaut. die änderung am description tag hat
funktioniert. außerdem habe ich noch ein paar andere fehler im <head>
korrigiert. ist alles in patch-1 gepushed. soll ich mergen oder willst
du das machen?

gruesse,
 gariepinus

On 2015-01-28 Wed 06:50 / 04, hexa- wrote:

> Hey,
> 
> kannst du mal versuchen das bei dir mit jekyll zu bauen und dann anzuschauen?
> 
> Gruß Martin
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/ChaosDarmstadt/chaos-darmstadt.de/pull/3#issuecomment-71846307

## 

  /\* Don't freak out */

mweinelt commented:
Wir sind keine Commit-Log-Fanatiker, mergen wir so. 

Danke!
